### ISSUE TITLE
flatmap-stream dependency is malicious

### ISSUE THREAD

**Issue by DokFaust** [tracking-exposed/facebook#97]
Currently we are depending on flatmap-stream, which contains a malicious payload that looks for crypto currencies wallets and tries to grab them [Link](https://www.npmjs.com/advisories/737).

The dependency on flatmap-stream is introduced by nodemon, which is on v1.8.3, but it seems that is has been fixed in v1.8.7.
So probably by dumping the version it should be fixed.

**Comment by DokFaust** [comment]
After updating `nodemon` it seems to work, but now `nconf` returns an error as it seems to not be able to receive the env settings:

```
 nconf loaded, using config/settings.json
/home/cclt/dev/facebook/app.js:33
    throw new Error("check your config/settings.json, config of 'interface' and 'post'
missing");
    ^

Error: check your config/settings.json, config of 'interface' and 'post' missing
    at Object.<anonymous> (/home/cclt/dev/facebook/app.js:33:11)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:774:12)
    at executeUserCode (internal/bootstrap/node.js:342:17)
    at startExecution (internal/bootstrap/node.js:276:5)
    at startup (internal/bootstrap/node.js:227:5)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by vecna** [comment]
whoa! thanks, but the solution is not dumping, rather on updating the version. right?

**Comment by DokFaust** [comment]
Yeah sorry, I meant dump as to "update to version" 1.8.7 of `nodemon` :D 
By doing so, `npm install` goes smoothly, but it crashes with the error listed above after `npm run watch`.
Now I dunno how the interface are passed to `nconf`, so I can't say why it happens.

**Comment by vecna** [comment]
one of the error you are experiencing depends on the fact that `config/settings.json` is not in .gitignore and the lack of this config file is not managed as an error. It should be addressed at startup safety check as #90

**Comment by vecna** [comment]
solved with 1.0.1 upcoming release
